tests/src/test/java/alluxio/shell/command/StatCommandTest.java

```diff
@@ -37,15 +37,15 @@ public void fileInfoNotExist() throws IOException {
   public void fileInfoWildCard() throws IOException, AlluxioException {
```

Comment:
done.
And directories with wildcard.
Also, I find a bug of stat or fileInfo command. see https://groups.google.com/forum/#!topic/alluxio-users/teg9KAh5rr8
I will create a new jira to fix it.